src/components/buttons.js

```diff
@@ -0,0 +1,76 @@
+      <button className="button" type="button">+/-</button>
+      <button className="button" type="button">%</button>
+      <button className="button operator" type="button">/</button>
+      <button className="button" type="button">7</button>
+      <button className="button" type="button">8</button>
+      <button className="button" type="button">9</button>
+      <button className="button operator" type="button">X</button>
+      <button className="button" type="button">4</button>
+      <button className="button" type="button">5</button>
+      <button className="button" type="button">6</button>
+      <button className="button operator" type="button">-</button>
+      <button className="button" type="button">1</button>
+      <button className="button" type="button">2</button>
+      <button className="button" type="button">3</button>
+      <button className="button operator" type="button">+</button>
+      <button className="button ziro" type="button">0</button>
+      <button className="button" type="button">.</button>
+      <button className="button operator" type="button">=</button>
+      {/* <button className="button equal" type="button">=</button> */}
+      {/* <div className="row">
+        <button className="button" type="button">AC</button>
+        <button className="button" type="button">+/-</button>
+        <button className="button" type="button">%</button>
+        <button className="button operator" type="button">/</button>
+        <button className="button" type="button">7</button>
+        <button className="button" type="button">8</button>
+        <button className="button" type="button">9</button>
+        <button className="button operator" type="button">X</button>
+      </div>
+      <div className="row">
+        <button className="button" type="button">7</button>
+        <button className="button" type="button">8</button>
+        <button className="button" type="button">9</button>
+        <button className="button operator" type="button">X</button>
+      </div>
+      <div className="row">
+        <button className="button" type="button">4</button>
+        <button className="button" type="button">5</button>
+        <button className="button" type="button">6</button>
+        <button className="button operator" type="button">-</button>
+      </div>
+      <div className="row">
+        <button className="button" type="button">1</button>
+        <button className="button" type="button">2</button>
+        <button className="button" type="button">3</button>
+        <button className="button operator" type="button">+</button>
+      </div>
+      <div className="row">
+        <button className="button ziro" type="button">0</button>
+        <button className="button" type="button">.</button>
+        <button className="button operator" type="button">=</button>
+        <button className="button equal" type="button">=</button>
+      </div> */}
```

Comment:
- Hey, please, kindly remove these commented pieces of code to make your codebase clean and maintainable.